worker/src/core/loaders/message-worker/index.ts

```diff
@@ -169,6 +170,45 @@ const finalize = tracer.wrap(
     const logGovsgJob = postmanConnection
       .query('SELECT log_next_job_govsg();')
       .then(([result]) => get(result, '[0].log_next_job_govsg', ''))
+      .then(async (campaignId: number) => {
+        const rawMessages = await postmanConnection.query(
+          "SELECT * FROM govsg_messages WHERE status = 'ERROR' AND campaign_id = :campaignId",
+          { type: QueryTypes.SELECT, replacements: { campaignId } }
+        )
+        if (!rawMessages) {
+          return campaignId
+        }
+
+        const messagesWithErr = rawMessages as Array<{
+          id: number
+          error_code: string
+          error_description: string
+        }>
+        if (
+          messagesWithErr.length === 0 ||
+          !config.get('sgcCampaignAlertChannelWebhookUrl')
+        ) {
+          return campaignId
+        }
+        let text = `*****NEW ALERT******\n*Campaign ${campaignId} has some errors*\n=====\n| Message ID | Error Code | Error Description |\n=====\n`
+        messagesWithErr.forEach((m) => {
+          text += `| ${m.id} | ${m.error_code} | ${m.error_description} |\n`
+        })
+        text += '***********'
+        await axios
+          .post(config.get('sgcCampaignAlertChannelWebhookUrl'), {
+            text: text,
+          })
+          .catch((e) => {
+            logger.error({
+              message: 'Failed to report errors to #sgc-campaign-alerts',
```

Comment:
This sounds like a never-ending cycle but:

Do we get alerted through other means when this happens?

Reply:
we could set up an alert for this alert lol, but the log agent could go down as well 😂

Reply:
Ok, this won't end well...

Let's keep it as-is.